docs/topics/caching-mission-intro.adoc

```diff
@@ -0,0 +1,39 @@
+Mission proficiency level: xref:proficiency_advanced[*{proficiency-advanced}*].
+
+The {name-mission-caching} mission demonstrates how to use a cache to increase the response time of applications.
+
+This mission shows you how to:
+
+* Deploy a cache to OpenShift.
+* Use a cache within an application.
+
+.About Caching
+Caching allows you to store information and access it for a given period of time.
+An advantage of caching is that a cache can provide access to information faster or more reliably than repeatedly calling the original service.
+A disadvantage of caching is that the cached information is not up to date but that can be mitigated by setting an _expiration_ or `TTL` (time to live) on each value stored in the cache.
+
+For example, assume you have two applications: _service1_ and _service2_. 
+If _service1_ depends on a value from _service2_ that infrequently changes, _service1_ could cache the value from _service2_ for a period of time. 
+If it takes _service1_ 500ms to retrieve the value directly from _service2_ but 100ms to retrieve the cached value, _service1_ would save 400ms by using the cached value for each cached call. 
+Using cached values can also reduce the number times _service2_ is called. 
+If _service1_ would make uncached calls to _service2_ 5 times per second, over 10 seconds, that would be 50 calls.
+If _service1_ started using a cached value with a `TTL` of 1 second instead, that would be reduced to 10 calls over 10 seconds.
+
+////
+For example, if _service1_ depends on a value from _service2_ that infrequently changes, _service1_ could cache the value from _service2_ for a period of time. 
+In the example with _service1_ and _service2_, if it takes _service1_ 500ms to retrieve the value from _service2_ but 100ms to retrieve the cached value, _service1_ would save 400ms by using the cached value for each cached call. 
+This can also reduce the number times _service2_ is called. 
+For example, if _service1_ would make uncached calls to _service2_ 5 times per second, over 10 seconds, that would be 50 calls.
+If _service1_ started using a cached value with a `TTL` of 1 second instead, that would be reduced to 10 calls over 10 seconds.
+////
+.How the {name-mission-caching} Mission Works
+. The _cache_, _name_, and _greeting_ services are deployed and exposed.
```

Comment:
I don't pretend to know why, but we don't call it "name", we call it "cute name".

Reply:
Fixed.